Dockerfile

```
@@ -0,0 +1,20 @@
+# Multi-stage build for optimized image size
+FROM eclipse-temurin:21-jre-alpine AS runtime
+
+# 작업 디렉토리 설정
+WORKDIR /app
+
+# JAR 파일 복사
+COPY build/libs/*.jar app.jar
+
+# 타임존 설정
+ENV TZ=Asia/Seoul
+RUN apk add --no-cache tzdata && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)